README.md

```diff
@@ -324,6 +335,8 @@ For example,
 a CSS shadow part for the close button,
 or some CSS variables.
 
+## Accessibility
+
```

Comment:
I'd add something like "TODO: determine detailed accessibility semantics; see #25 for initial discussion."

Reply:
Done.